src/account-type-utils.js

```diff
@@ -42,4 +42,37 @@ const getAccountType = email => {
   ) || 'Unknown'
 }
 
-module.exports = { getAccountType }
+/**
+ * Retrieves the domain name from the given email address and returns it as a string.
+ *
+ * @param {string} email The email address to retrieve the domain from.
+ * @returns {string} Returns the domain name or 'Unknown' if no domain name is found.
+ * @example
+ *
+ * getEmailDomain('johndoe@example.com')
+ * // => 'example.com'
+ *
+ * getEmailDomain('johndoe.example.com')
+ * // => 'Unknown'
+ *
+ * getEmailDomain('johndoe@example')
+ * // => 'Unknown'
+ *
+ */
+const getEmailDomain = email => {
+  if (!_.includes(email, '@') ||
```

Comment:
nit: we could emphasize the posititive case and return unknown as as the default. Something like:

```javascript
return _.includes(email, '@') && _.includes(email, '.') 
    ? _.chain(email).split('@').last() 
    : 'Unknown'
```

If you choose to do this, please double check the syntax as I have not used chain or the non-fp variant of lodash very much.